.circleci/config.yml

```diff
@@ -681,6 +669,17 @@ jobs:
             cd "$PROTOCOL_DIR/solana-programs/anchor/audius-data"
             source /opt/circleci/.nvm/nvm.sh
             npm run install-dev
+      - run:
+          name: anchor tests
+          command: |
+            export PROTOCOL_DIR="/home/circleci/project"
+            export PATH="/home/circleci/.local/share/solana/install/active_release/bin:/usr/local/lbin:$PATH"
+            cd "$PROTOCOL_DIR/solana-programs/anchor/audius-data"
+            npm test
```

Comment:
why are we using `npm test`? https://project-serum.github.io/anchor/cli/commands.html#test

Reply:
https://github.com/AudiusProject/audius-protocol/blob/master/.circleci/config.yml#L616 this (master) makes sense

Reply:
we're running the same npm test as locally - to run locally you need to `export BROWSER=` and this standardizes - we are using npm scripts as the single entrypoint for all audius-data stuff - `npm run install-dev`, `npm run build-dev`, `npm run deploy-dev`. we follow this convention elsewhere as well where node projects have `npm lint` and other such scripts wrapping whatever other tooling we're using - so this made sense to do. you can easily look up that it's using `anchor test` under the hood by checking the package.json